README.md

```diff
@@ -1,8 +1,31 @@
-# binding-tariff-classification
+# Binding Tariff Classification
 
 [![Build Status](https://travis-ci.org/hmrc/binding-tariff-classification.svg)](https://travis-ci.org/hmrc/binding-tariff-classification) [ ![Download](https://api.bintray.com/packages/hmrc/releases/binding-tariff-classification/images/download.svg) ](https://bintray.com/hmrc/releases/binding-tariff-classification/_latestVersion)
 
-This is a placeholder README.md for a new repository
+This is the Back End for the Binding Tariff Suite of applications e.g.
+
+- [BTI Application Form](https://github.com/hmrc/tariff-classification-frontend) on GOV.UK
```

Comment:
The two items here have the wrong links. You need to swap them

Reply:
Not sure what you're referring to